router.js

```diff
@@ -75,7 +75,7 @@ exports.setup = function() {
 
   app.use(
     "/xsl",
-    express.static(path.join(__dirname, "client"), {
+    express.static(path.join(__dirname, "client/xsl"), {
```

Comment:
please change this to path.join(__dirname, "client", "xsl")

For cross-platform functionality